lib/signs_ui/signs/sign.ex

```diff
@@ -49,6 +50,21 @@ defmodule SignsUI.Signs.Sign do
     %__MODULE__{id: id, config: %{mode: :off, expires: nil}}
   end
 
+  @spec expiration_from_string(String.t() | nil) :: expires_on()
+  defp expiration_from_string(expiration_string)
+       when not is_nil(expiration_string) and expiration_string != "" do
```

Comment:
I think this can be just `is_binary(expiration_string)`. A value of `""` is just as invalid as `"not-a-datetime"` and should trigger the same warning, IMO.

Reply:
Done